### PR TITLE
Basic behaviour

### PR DESCRIPTION
**Pull request by alicebartlett**
Still quite a lot to do on this, but here's a PR for the code to stick the thing to the page as the user scrolls.

### CODE REVIEW

File: src/js/inPageNav.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+		/* which menu item should be highlighted */
+		return false;
+	}
+
+	shouldDock(){
+		const scrolltop = window.pageYOffset || document.body.scrollTop;
+		const dockpoint = InPageNav.offset(this.inPageNavEl) + this.inPageNavEl.scrollHeight;
+
+		return scrolltop > dockpoint
```
Comment (onishiweb) [inline src/js/inPageNav.js:48]: semi-colon missing eol

File: src/js/nav.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
+/**
```
Comment (onishiweb) [inline src/js/nav.js:1]: Is this meant to be check this in as part of the PR? Or is it just something you're working through removing?

**Review by onishiweb — COMMENTED** [review]
Just a couple of things to check, otherwise looks good.

File: src/js/nav.js (hunk 2 above)
Comment (alicebartlett) [reply src/js/nav.js:1]: This is just placeholder cruft to help me remember to add stuff. A better me would do some line-by-line commits so I can omit this but... I'm the me you have so ¯\\\_(ツ)_/¯

**Pull request merged**